test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -219,12 +241,47 @@ module.exports = {
     }
 
     return {
-      getPactified: () => {
-        return pactRegister.pactify(data)
-      },
-      getPlain: () => {
-        return data
+      getPactified: () => pactRegister.pactify(data),
+      getPlain: () => data
+    }
+  },
+  validTransactionEventsResponse: (opts = {}) => {
+    const data = {
+      transaction_id: opts.transaction_id || 'ht439nfg2l1e303k0dmifrn4fc',
+      events: (opts.payment_states) ? buildPaymentEvents(opts) : buildChargeEventStateWithDefaults(opts)
+    }
+
+    return {
+      getPactified: () => pactRegister.pactify(data),
+      getPlain: () => data
+    }
+  },
+  validTransactionSearchResponse: (opts = {}) => {
+    let results = []
```

Comment:
`transactionSearchResult` or something more meaningful would be better.

Reply:
Everything is called either xxxRequest or xxxResponse thus I believe it'd confusing naming this anything else